.github/workflows/private_fork_scheduled_codebuild.yml

```diff
@@ -0,0 +1,73 @@
+---
+name: s2nPrivateFuzzScheduled
+
+on:
+  schedule:
+    - cron: '0 13 * * *'
+jobs:
+  fuzz:
+    if: startsWith(github.event.repository.name, 'private-')
+    runs-on: ubuntu-18.04
+    strategy:
+      matrix:
+        openssl_version:
+          - openssl-1.1.1
+        specific_test:
+          - s2n_bike_r1_fuzz_test
+          - s2n_bike_r2_fuzz_test
+          - s2n_certificate_extensions_parse_test
+          - s2n_client_cert_recv_test
+          - s2n_client_cert_req_recv_test
+          - s2n_client_cert_verify_recv_test
+          - s2n_client_fuzz_test
+          - s2n_client_hello_recv_fuzz_test
+          - s2n_client_key_recv_fuzz_test
+          - s2n_encrypted_extensions_recv_test
+          - s2n_extensions_client_key_share_recv_test
+          - s2n_extensions_client_supported_versions_recv_test
+          - s2n_extensions_server_key_share_recv_test
+          - s2n_extensions_server_supported_versions_recv_test
+          - s2n_hybrid_ecdhe_bike_r1_fuzz_test
+          - s2n_hybrid_ecdhe_bike_r2_fuzz_test
+          - s2n_hybrid_ecdhe_sike_r1_fuzz_test
+          - s2n_hybrid_ecdhe_sike_r2_fuzz_test
+          - s2n_memory_leak_negative_test
+          - s2n_openssl_diff_pem_parsing_test
+          - s2n_recv_client_supported_groups_test
+          - s2n_select_server_cert_test
+          - s2n_server_cert_recv_test
+          - s2n_server_extensions_recv_test
+          - s2n_server_fuzz_test
+          - s2n_server_hello_recv_test
+          - s2n_sike_r1_fuzz_test
+          - s2n_sike_r2_fuzz_test
+          - s2n_stuffer_pem_fuzz_test
```

Comment:
do we need to update this list whenever we add a new test ?

Reply:
Yes, that would be a downside to this approach...or we could generate this list in some automated way (Makefile step).